tests/unittests/test_core_stream_query_building.py

```diff
@@ -0,0 +1,70 @@
+import unittest
+from tap_google_ads.streams import create_core_stream_query
+
+SELECTED_FIELDS = ["id"]
+RESOURCE_NAME = "ads"
+
+class TestFullTableQuery(unittest.TestCase):
+    """
+    Test that `create_core_stream_query` function build appropriate query with WHERE, ORDER BY clause.
+    """
+    def test_empty_filter_params_clause(self):
+        """
+        Verify that query does not contain WHERE and ORDER BY clause if filter_params value is None.
+        """
+
+        filter_params = None
+        last_pk_fetched = {}
+        composite_pks = False
+
+        expected = 'SELECT id FROM ads  PARAMETERS omit_unselected_resource_names=true'
+
+        actual = create_core_stream_query(RESOURCE_NAME, SELECTED_FIELDS, last_pk_fetched, filter_params, composite_pks)
```

Comment:
Replace these `expected` and `actual` with `expected_query`  and `actual_query` in all the test cases.

Reply:
Replaced `expected` and `actual` with `expected_query` and `actual_query`.